### DISCUSSION TITLE
Fyrtur power supply

### DISCUSSION THREAD

**Discussion by illivanillie** [Koenkk/zigbee2mqtt#8571] [Devices]
If I connect a power supply to ikea fyrtur (or any other battery powered device), will it recognize that it is connected to AC/power supply and behave differently? I want to prevent lags and deepsleep

greetings

**Comment by sjorge** [comment]
No, it will still act as an end device, not a router.

**Reply by illivanillie** [reply]
It doesn’t need to be a router by definition.
So there will be zero enhancements?

**Reply by sjorge** [reply]
Nothing will change if you just swap the battery for a constant power supply, the firmware will still act like an end device and sleep and periodically check for messages.
